infra/modules/database/instance.tf

```diff
@@ -0,0 +1,40 @@
+# DMS replication instance and endpoint connections
```

Comment:
⭐⭐  &nbsp; _(optional but seriously consider)_ renaming the file to `dms_instance.tf` or similar